src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1355,14 +1356,18 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         $classToServices = [
             LocoProviderFactory::class => 'translation.provider_factory.loco',
+            LokaliseProviderFactory::class => 'translation.provider_factory.lokalise',
         ];
 
         $parentPackages = ['symfony/framework-bundle', 'symfony/translation', 'symfony/http-client'];
 
         foreach ($classToServices as $class => $service) {
-            $package = sprintf('symfony/%s-translation', substr($service, \strlen('translation.provider_factory.')));
+            switch ($package = substr($service, \strlen('translation.provider_factory.'))) {
+                case 'loco': $package = 'loco'; break;
+                case 'lokalise': $package = 'lokalise'; break;
```

Comment:
We use this code part in notifier too, but it is only needed if we want to go from: microsoftteams to microsoft-teams

Imho you can remove these

Reply:
Yes, at the end if we don't need to fix the name, `$package === $package`. I will fix it. @andrii-bodnar could you do the same thing for Crowdin Provider? Thanks